setup.py

```diff
@@ -0,0 +1,50 @@
+from setuptools import setup
+from setuptools import find_packages
+
+with open("README.md", "r") as f:
+    readme = f.read()
+
+with open("VERSION", "r") as f:
+    version = f.read().strip()
+
+
+install_requires = ["dask", "distributed", "torch", "pillow"]
+testing_deps = ["black", "pytest", "pytest-cov", "flake8", "mypy"]
+
+
+setup(
+    name="dask-pytorch",
+    version=version,
+    maintainer="Saturn Cloud Developers",
+    maintainer_email="open-source@saturncloud.io",
+    license="BSD 3-clause",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "License :: OSI Approved :: BSD License",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
```

Comment:
```suggestion
        "Intended Audience :: Science/Research",
        "Topic :: Scientific/Engineering :: Image Processing",
        "Topic :: Scientific/Engineering :: Image Recognition",
```

I think these other two image-y classifiers from https://pypi.org/classifiers/ are also appropriate for this library